tests/functional/adapter/utils/test_utils.py

```diff
@@ -1,4 +1,9 @@
 import pytest
+from tests.functional.adapter.fixture_datediff import (
```

Comment:
This should not be normal. Project dependencies should be after standard and 3rd party imports no ?

Reply:
Correct, this is indeed weird,  as it's a project deps.

Reply:
odd, there is something that leads `pre-commit` to follow another order, maybe the fact that `from pkgutil import extend_path` was added?

Reply:
Good catch ;)